runner.py

```diff
@@ -15,28 +15,29 @@ def inner():
         return res
     return inner
 
-move = _refresh(_board.move)
-turn_left = _refresh(_board.turn_left)
-pick_beeper = _refresh(_board.pick_beeper)
-put_beeper = _refresh(_board.put_beeper)
-front_is_clear = _board.front_is_clear
-right_is_clear = _board.right_is_clear
-left_is_clear = _board.left_is_clear
-beeper_is_present = _board.beeper_is_present
-facing_north = _board.facing_north
-facing_south = _board.facing_south
-facing_east = _board.facing_east
-facing_west = _board.facing_west
-
 def execute(callback):
     try:
         _board.start_screen()
         _board.draw()
```

Comment:
Since `execute` gets called multiple times now, I think we should lift these two lines, the calls to `.start_screen` and `.draw`, to right after we instantiate `_board`. We already call `.draw` in `_refresh`, and `.start_screen` should only get invoked once near the beginning of running.